h2o-r/demos/rdemo.coxph.R

```diff
@@ -0,0 +1,86 @@
+# Load the libraries used to analyze the data
+library(survival)
+library(MASS)
+library(h2o)
+
+# Start H2O cluster
+h2o.init()
+
+# Get path to data file
+locate_source <- function(file) {
+  file_path <- try(h2o:::.h2o.locate(file), silent = TRUE)
+  if (inherits(file_path, "try-error")) {
+    file_path <- paste0("s3n://h2o-public-test-data/", file)
```

Comment:
Are you using s3n because it doesn't require credentials? We are trying to discourage users from using s3n to import data, s3 should be faster. Perhaps we can change it to `http` instead? Would be better not to show importing from s3n in a demo.